src/map/battle.cpp

```diff
@@ -2270,7 +2272,11 @@ static int battle_calc_base_weapon_attack(struct block_list *src, struct status_
 		atkmin = max(0, (int)(atkmin - variance + base_stat_bonus));
 		atkmax = min(UINT16_MAX, (int)(atkmax + variance + base_stat_bonus));
 
+#ifdef RENEWAL
+		if ((sc && sc->data[SC_MAXIMIZEPOWER]) || critical == true)
+#else
 		if (sc && sc->data[SC_MAXIMIZEPOWER])
+#endif
```

Comment:
```suggestion
		if ((sc && sc->data[SC_MAXIMIZEPOWER]) || critical == true)
```
Pre-re also uses maxAtk for critical hits